app/exporters/hathi/image_metadata.rb

```diff
@@ -2,47 +2,49 @@
 require "mini_magick"
 
 module Hathi
+  # Extrapolates image data for entire resource from the first image
   class ImageMetadata
-    attr_reader :resource, :repr_image
+    attr_reader :resource, :original_image
     def initialize(resource:)
       @resource = resource
       wayfinder = Wayfinder.for(resource)
-      page = wayfinder.members.first
-      metadata = page.original_file
-      tiff_file = Valkyrie::StorageAdapter.find_by(id: metadata.file_identifiers.first)
-      @repr_image = MiniMagick::Image.new(tiff_file.disk_path)
+      page = ContentPackage::Page.new(wayfinder.members.first, "representative page")
+      # page = wayfinder.members.first
+
+      @original_image = MiniMagick::Image.new(page.tiff_path)
+      @derivative_image = MiniMagick::Image.new(page.derivative_path)
     end
 
     def capture_date
-      repr_image.data["properties"]["tiff:timestamp"]
+      original_image.data["properties"]["xmp:CreateDate"]
     end
 
     def scanner_make
-      repr_image.data["properties"]["tiff:make"]
+      original_image.data["properties"]["tiff:make"]
     end
 
     def scanner_model
-      repr_image.data["properties"]["tiff:model"]
+      original_image.data["properties"]["tiff:model"]
     end
 
     def scanner_user
-      "Princeton University Library: Digital Photography Studio"
+      %("Princeton University Library: Digital Photography Studio")
     end
 
     def bitonal?
-      repr_image["%z"] == 1
+      original_image["%z"] == 1
     end
 
     def resolution
-      repr_image["resolution"].first
+      original_image["resolution"].first
     end
 
     def scanning_order
-      "left_to_right"
```

Comment:
We do have some right-to-left items, so maybe checking the `viewing_direction` property would be good here?

Reply:
Here is Hathi's sample meta.yaml file: https://drive.google.com/file/d/0B0EHs5JWGUMLWjU2OHVhQzN5WEk/view